### PR TITLE
feat: prebuilt binaries with Ledger support

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/2729

This PR removes [checksum signing](https://goreleaser.com/customization/sign/) because https://github.com/goreleaser/goreleaser-cross/issues/58. 

## Testing

The assets on https://github.com/rootulp/celestia-app/releases/tag/v1.0.0-rc52

OS | Platform | Ledger support 
--- | --- | ---
Darwin | arm64 | ✅  
Darwin | amd64 | ✅ 
Linux | arm64 | 
Linux | amd64 | ✅

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -1,36 +1,93 @@
...
+# for multiple platforms. See https://github.com/goreleaser/goreleaser-cross
 before:
   hooks:
     - go mod tidy
 builds:
-  - main: ./cmd/celestia-appd
+  - id: darwin-amd64
+    main: ./cmd/celestia-appd
     binary: celestia-appd
     env:
```
[the remark this answers is not in the document]
Comment (rootulp) [reply .goreleaser.yaml:12]: I don't believe this suggestion b/c it works as-is.

**Review by evan-forbes — APPROVED** [review]
what does the smiley face mean for the linux entry? do you need a volunteer to try it?

**Comment by cmwaters** [comment]
> I get [this error](https://pastebin.com/8dAkHE17) instead of an error claiming Ledger is unsupported.

Are you sure you had your ledger unlocked and with Cosmos app open. I got the same but then it worked once I unlocked my ledger

**Review by cmwaters — APPROVED** [review]

**Comment by rootulp** [comment]
I'll hold off on merging until @evan-forbes tests on Linux. You should be able to test via:

```
./celestia-appd keys add ledger-1 --ledger
```

**Review by evan-forbes — APPROVED** [review]

**Comment by rootulp** [comment]
Out of curiosity which architecture is your Linux machine? arm64 or amd64?

**Review by cmwaters — APPROVED** [review]
Reapproving

**Pull request merged**